### PR TITLE
[docs] move documentation to website

### PR DESCRIPTION
**Pull request by JanKoehnlein**
https://github.com/theia-ide/theia-website

### CODE REVIEW

**Comment by akosyakov** [comment]
I am concerned about moving away development docs.

**Comment by kittaakos** [comment]
> I am concerned about moving away development docs.

Too early upvote from me... Eventually, everything is [there](https://github.com/theia-ide/theia/tree/407d5d6e28327c25f43d7511094476730fb79c17) (except the [`Develop a new Theia extension `](http://www.theia-ide.org/doc/Authoring_Extensions.md)), and the [new documentation](http://www.theia-ide.org/doc/) looks way better.

File: README.md [hunk 1]
```diff
@@ -27,30 +31,16 @@ Theia is an extensible platform to develop full-fledged multi-language Cloud & D
...
-  - [**Events**](doc/Internals.md#events)
-  - [**Commands**](doc/Commands_Keybindings.md)
-  - [**Preferences**](doc/Preferences.md)
-
 ## Contributing
 
 Read below to learn how to take part in improving Theia:
-- Fork the repository and [run the examples from source](doc/Developing.md#quick-start)
-- Get familiar with [the development workflow](doc/Developing.md), [Coding Guidelines](https://github.com/theia-ide/theia/wiki/Coding-Guidelines), [Code of Conduct](CODE_OF_CONDUCT.md) and [learn how to sign your work](CONTRIBUTING.md#sign-your-work)
+- Fork the repository and [run the examples from source](http://www.theia-ide.org/doc/Developing.md#quick-start)
```
Comment (akosyakov) [inline README.md]: links seem to be broken

**Comment by svenefftinge** [comment]
I agree with Anton, that the information about contributing / developing on the theia project should stay here. The documentation is for downstream extenders / users not for contributors.

**Comment by JanKoehnlein** [comment]
Done. Should the contributor doc be remove from the website?

**Comment by svenefftinge** [comment]
> Should the contributor doc be remove from the website?

Yes

**Pull request merged**